webapp/packages/plugin-sql-editor-navigation-tab/src/SqlEditorTabService.ts

```diff
@@ -309,6 +309,7 @@ export class SqlEditorTabService extends Bootstrap {
     tab.handlerState.resultTabs = observable([]);
     tab.handlerState.executionPlanTabs = observable([]);
     tab.handlerState.statisticsTabs = observable([]);
+    tab.handlerState.outputLogsTab = undefined;
```

Comment:
you can remove this line because other tabs are reset because they can't restore full state after page refresh, but this tab stores all its state